token/services/identity/role/role.go

```diff
@@ -101,48 +101,48 @@ func (r *Role) mapStringToID(ctx context.Context, v string) (driver.Identity, st
 	labelAsIdentity := driver.Identity(label)
 
 	// check immediately if there is an identifier with that label
-	if idIdentifier, err := r.localMembership.GetIdentifier(labelAsIdentity); err == nil {
+	if idIdentifier, err := r.localMembership.GetIdentifier(ctx, labelAsIdentity); err == nil {
 		return nil, idIdentifier, nil
 	}
 
 	switch {
 	case len(label) == 0:
-		r.logger.Debugf("passed empty label")
+		r.logger.DebugfContext(ctx, "passed empty label")
 		return nil, defaultIdentifier, nil
 	case label == defaultIdentifier:
-		r.logger.Debugf("passed default identifier")
+		r.logger.DebugfContext(ctx, "passed default identifier")
 		return nil, defaultIdentifier, nil
 	case label == defaultNetworkIdentity.UniqueID():
-		r.logger.Debugf("passed default identity")
+		r.logger.DebugfContext(ctx, "passed default identity")
```

Comment:
I am wondering if these logs really must be traces. 

Do you have a strategy in mind when something is logged vs traced+logged?

I have the fear that adding more and more traces will make it difficult to read.